shared/cache.ts

```diff
@@ -0,0 +1,61 @@
+import { DateTime } from "luxon";
+
+type Cache<DataType> = {
+  value?: DataType;
+  populatedAt?: string;
+  ttl: number;
```

Comment:
thoughts on saying `ttlInSeconds` so peeps know this is in seconds not mins or hours. 

Not too fussed tbh so feel free to ignore.

Reply:
`timeToLiveInSeconds` 😅 

Happy to update ;)